types/tests/styled-components.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+import tw from '..'
+
+export const Container = styled.div`
+  ${tw`flex flex-col w-full max-w-sm`}
+  & > form {
+    ${tw`w-full`}
+  }
+`
+
+// I want to test the styled-components css prop in isolation from the emotion css prop,
+// but I dunno how to do that
+// export const cssProp = <div css={tw`bg-red-100`} />
```

Comment:
wouldn't this prop be taken from the import of `styled-components`? 

I assume that the emotion `css` prop is only injected on files processed by emotion's `jsx`

Reply:
Emotion's `css` prop is a global type declaration on the `JSX` namespace. Uncomment that line, then ctrl+click on `css`, and you'll see it leads to emotion's types, not SC's, and I'm not sure how to isolate the global typings

Reply:
Strike that, it looks like I just needed to put each test in their own folder with their own tsconfig. Kind of noisy, but eh, it works